src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -19,8 +19,10 @@ import { IQuery } from '../../rest/Query';
 import { IQueryResults } from '../../rest/QueryResults';
 
 export interface IQuerySummaryOptions {
+  enableResultsSummary?: boolean;
+  enableNoResultsForSearch?: boolean;
+  enableCancelLastAction?: boolean;
   enableSearchTips?: boolean;
-  onlyDisplaySearchTips?: boolean;
```

Comment:
We cannot remove options like this, as this is a breaking change in the component API.

The same component configuration with the same option should yield the same resulting behaviour after the change. We can add new option, we can mark some older one as "deprecated", but we cannot remove option, or modify how they behave in a non major/breaking version.

We have a mechanism in place to rename option if needed.

Reply:
It totally makes sense! I will make the adjustments!